tracer/test/Datadog.Trace.Tools.dd_dotnet.ArtifactTests/CreatedumpTests.cs

```diff
@@ -371,6 +371,28 @@ public async Task NativeCrash()
         File.Exists(reportFile.Path).Should().BeTrue();
     }
 
+    [SkippableFact]
+    public async Task ResumeProcessWhenCrashing()
+    {
+        SkipOn.Platform(SkipOn.PlatformValue.MacOs);
+        SkipOn.PlatformAndArchitecture(SkipOn.PlatformValue.Windows, SkipOn.ArchitectureValue.X86);
```

Comment:
So this works on Linux then too? Even though `kernel32.dll`?

Reply:
On Linux we should have the same requirements (= the crashing process shouldn't remain frozen if dd-dotnet crashes). It just happens that we don't need `SetUnhandledExceptionFilter` for that, Linux automatically resumes the suspended threads when dd-dotnet crashes.

You may then think "but wait, why are the installer tests failing then?". Yup, I know right?